Hodge_Zach/assignments/great_number_game/templates/win.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>You Win!</title>
+    </head>
+    <body>
+        <h1>Welcome to the Great Number Game!</h1>
+
+        <h3>I am thinking of a number between 1 and 100.</h3>
+
+        <div class="box">
+            <h2>{{randnum}} was the number!</h2>
+        </div>
+
+        <form action="/goback" method="post">
+            <input type="submit" name="submit" value="Play Again!">
+        </form>
+    </body>
+</html>
```

Comment:
So this works.

But the way you programmed it would make very difficult on you + other developers. Remember my analogy on 'does google hire someone to create 1 page for every map direction, e.g. from Dallas TX to the Great Wall of China?'

Rather than creating 3 views, use just 1 single view that you modify dynamically based off of the game state. let's see what you can do 😎